app/Http/Controllers/Api/V1/GoogleClassroomController.php

```diff
@@ -565,8 +556,8 @@ public function getCourseTopics(Request $request)
      * @urlParam project required The Id of a project. Example: 9
      * @urlParam playlist required The Id of a playlist. Example: 10
      * @bodyParam access_token string|null The stringified of the GAPI access token JSON object
-     * @bodyParam string course_id (The Google Classroom course id)
-     * @bodyParam string topic_id (The Google Classroom topic id)
+     * @bodyParam course_id string The Google Classroom course id
```

Comment:
Add "Example:" for "course_id" and "topic_id".

Reply:
Done!